demo/src/v3/java/com/google/maps/android/utils/demo/BaseDemoActivity.java

```diff
@@ -1,6 +1,6 @@
 /**
  * DO NOT EDIT THIS FILE.
- * <p>
+ *
```

Comment:
I believe if you rebase this it should get rid of a lot of these formatting changes in the v3 product flavor, given that https://github.com/googlemaps/android-maps-utils/pull/896 has been merged.